### PR TITLE
Regina

### PR DESCRIPTION
**Pull request by PalatkinaRegina**
Asks you to accept her branch

### CODE REVIEW

File: README.md
Comment (mikevp777) [inline README.md:1]: Зачем

**Review by mikevp777 — CHANGES_REQUESTED** [review]
Write my name instead of yours

**Comment by PalatkinaRegina** [comment]
added your name at commit 597f0074784d4fb6f60ade5be5812060ecb627f2

**Review by mikevp777 — APPROVED** [review]
It's ok

**Pull request merged**